### DISCUSSION TITLE
PD-1 Staining

### DISCUSSION THREAD

**Discussion by ifeanyianidi** [IBEXImagingCommunity/ibex_imaging_knowledge_base#78] [Q&A]
Hi all,
Does this look like appropriate PD-1 staining in the attached images of a lymph node? 
I used the PD1 AF555 antibody (Vendor: Abcam, Clone: EPR4877, Cat #: Ab275126).
CD20 staining is in magenta, PD-1 staining is in yellow. 

Thank you,
Ifeanyi
<img width="1608" alt="PD1 CD20 LN" src="https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/128619454/7e141777-d18d-47e8-874e-95eba7a71285">

**Comment by anita-gola** [comment]
Hi Ifeanyi, 
thanks for sharing your image. Do you mind sending us a more zoomed-in image of the PD1? PD1 should result in clear membrane staining in the Lymph node, and it is not so easy to determine this from your images. Additionally, could you confirm with us the origin of the tissue (ie. mouse vs human for example?). Amongst the many papers that have imaged PD1, I would like to suggest these ones for you to take a look as potential references: https://www.sciencedirect.com/science/article/pii/S0092867421006589?via%3Dihub, https://www.pnas.org/doi/10.1073/pnas.2018488117. 
Good luck!

**Comment by radtkea** [comment]
Hi Ifeanyi,

I agree with Anita. It's hard to see from the image but I would guess not?

This is the same antibody you are using but we are getting better labeling of T  follicular helper cells in the germinal centers of tonsil samples. You will see in my notes that I recommend a 1:10 dilution for best results and do dual antigen retrieval (pH 6 and pH 9).
https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/supporting_material/PD-1_AF555/0000-0003-4379-8967.html

Best wishes,
A

**Comment by Katitamorais** [comment]
Hello!
@ifeanyianidi, could you share which CD20 clone you are using? FFPE or frozen tissue?
Thanks.
